QWeb/internal/table.py

```diff
@@ -193,7 +198,8 @@ def get_clickable_cell(self,
             return clickable_child[int(index) - 1]
         return table_cell
 
-    def get_cell_by_locator(self, locator: str) -> int:
+    def get_cell_by_locator(self, locator: str, **kwargs) -> int:
+        partial_match = util.par2bool(kwargs.get('partial_match', CONFIG['PartialMatch']))
```

Comment:
Shouldn't this also check if `partial_match` was provided in the kwargs?

Reply:
wdym? It get's partial_match from kwargs if it's given, if not it takes it from CONFIG

Reply:
I don't know what I've been thinking 😅 
My bad, resolving comment.